cryptoinvestor/templates/assets/assets.html

```diff
@@ -19,6 +27,14 @@
                     <td>{{ asset.id }}</td>
                     <td>{{ asset.name }}</td>
                     <td>{{ asset.rate }}</td>
+                    <td> 
+                        <input id="{{asset.name}}" type="number" data-length="6">
+                        <label for="input_text">Cnt:</label>   
+                    </td>
+                    <td> 
+                        <a id="buy_btn" class="waves-effect waves-light btn-small"  onclick="get('buy',{{asset}})" >Buy</a> 
```

Comment:
you are creating multiple buttons with the same `id`. Please use for cycle counter in jinja2 if you need the id

Reply:
done